acoustid/future/fpindex/client.py

```
@@ -0,0 +1,282 @@
+        Returns:
+            True if index exists, False otherwise
+        """
+        try:
+            status, _ = await self._request(
+                "HEAD", f"/{index_name}", expected_status=[200, 404]
+            )
+            return status == 200
+        except FingerprintIndexClientError:
+            return False
+
+    async def get_index_info(self, index_name: str) -> Dict:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)